bitmark-references/bitmark-sdk/account.md

```diff
@@ -0,0 +1,280 @@
+# Account
+
+Within the Bitmark system, an account represents any entity capable of creating and owning property, whether individuals, institutions, or organizations.
+
+An account incorporates the public-private key-pair and the private key is required to [digitally sign](https://en.wikipedia.org/wiki/Digital_signature) any Bitmark blockchain record, including asset records, issue records, and transfer records.
+
+## Create an account
+
```

Comment:
`bitmark-sdk-document.md` is going to be the fist page for sure.

As for a short description for this section, there is a section in the beginning of this page explaining what is an account, so it could be redundant to add the description.

Possible improvements:
1. Add the desc to `## Create an account`: `Create a new identity to participate Bitmark blockchain.`
2. Merge the two sections: `# Account` and `## Create an account`